<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMLTemplateDialog</class>
 <widget class="QWidget" name="UMLTemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>General Properties</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="UMLDatatypeWidget" name="dataTypeWidget" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="tb_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="UMLStereotypeWidget" name="stereotypeWidget" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="DocumentationWidget" name="documentationWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UMLDatatypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umldatatypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UMLStereotypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umlstereotypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DocumentationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/documentationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
